doc/ansible/dev/developer_guide.md

```diff
@@ -378,10 +382,13 @@ from your application, then simply update the watches file with `manageStatus`:
   manageStatus: false
 ```
 
-To update the `status` subresource with key `foo` and value `bar`, `k8s_status`
-can be used as shown:
+The simplest way to invoke the `k8s_status` module is to
+use its fully qualified collection name (fqcn). To update the
+`status` subresource with key `foo` and value `bar`, `k8s_status` can be
+used as shown:
+
 ```yaml
-- k8s_status:
+- operator_sdk.util.k8s_status:
```

Comment:
It shows a break change. Then, in POV this info needs to be signalized in the CHANGELOG.